examples/notebooks/multi_user.ipynb

```diff
@@ -0,0 +1,161 @@
+{
+  "cells": [
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "4pmDE-zGf_CO"
+      },
+      "outputs": [],
+      "source": [
+        "import os\n",
+        "import time\n",
+        "\n",
+        "import numpy as np\n",
+        "import pandas as pd\n",
+        "import matplotlib.pyplot as plt\n",
+        "\n",
+        "from elfpy.simulators import YieldSimulator\n",
+        "from elfpy.utils.data import format_trades"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# make folder if it doesn't exit\n",
+        "out_dir = os.path.join(os.pardir, 'outputs')\n",
+        "if not os.path.exists(out_dir):\n",
+        "    os.makedirs(out_dir)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "tfTl4-SGBCi9"
+      },
+      "outputs": [],
+      "source": [
+        "from collections import UserString\n",
+        "\n",
+        "\n",
+        "pd.set_option(\"float_format\",'{:,.6f}'.format)\n",
+        "\n",
+        "random_seed = 3\n",
+        "simulator_rng = np.random.default_rng(random_seed)\n",
+        "config = {\n",
+        "    \"min_fee\": 0.1, # decimal that assigns fee_percent\n",
+        "    \"max_fee\": 0.5, # decimal that assigns fee_percent\n",
+        "    \"min_target_liquidity\": 1e6, # in USD\n",
+        "    \"max_target_liquidity\": 10e6, # in USD\n",
+        "    \"min_target_volume\": 0.001, # fraction of pool liquidity\n",
+        "    \"max_target_volume\": 0.01, # fration of pool liquidity\n",
+        "    \"min_pool_apy\": 0.02, # as a decimal\n",
+        "    \"max_pool_apy\": 0.9, # as a decimal\n",
+        "    \"min_vault_age\": 0, # fraction of a year\n",
+        "    \"max_vault_age\": 1, # fraction of a year\n",
+        "    \"min_vault_apy\": 0.001, # as a decimal\n",
+        "    \"max_vault_apy\": 0.9, # as a decimal\n",
+        "    \"base_asset_price\": 2.5e3, # aka market price\n",
+        "    \"pool_duration\": 180, # in days\n",
+        "    \"num_trading_days\": 180, # should be <= pool_duration\n",
+        "    \"floor_fee\": 0, # minimum fee percentage (bps)\n",
+        "    \"tokens\": [\"base\", \"fyt\"],\n",
+        "    \"trade_direction\": \"out\",\n",
+        "    \"precision\": None,\n",
+        "    \"pricing_model_name\": \"Element\",\n",
+        "    \"rng\": simulator_rng,\n",
+        "    \"verbose\": False,\n",
+        "}\n",
+        "\n",
+        "config[\"user_policy\"] = {\n",
+        "    \"trade\": {\n",
+        "        \"conditional\": {\n",
+        "            \"if\": {\n",
+        "                (\"gt\", \"market.apy\", 0.5)\n",
+        "            },\n",
+        "            \"then\": \"buy\"\n",
```

Comment:
missing comma. Update to:
        "            \"then\": \"buy\",\n",

Reply:
removed this notebook -- it's dumb.